wpilibj/src/main/java/edu/wpi/first/wpilibj/Preferences.java

```diff
@@ -173,6 +173,13 @@ public void remove(String key) {
     m_table.delete(key);
   }
 
+  /**
+   * Remove all preferences.
+   */
+  public void removeAll() {
+    m_table.getKeys().forEach(this::remove);
```

Comment:
This doesn't throw a ConcurrentModificationException?

Reply:
getKeys returns a new set so it shouldn't